omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -91,7 +107,15 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 		
 		T delegate = handler.newDelegate();
 		setConvertedProperties(delegate, propertiesToCreate, handler.getCreatableProperties(), true);
-		ValidateUtil.validate(delegate);
+		
+		try {
+			ValidateUtil.validate(delegate);

```

Comment:
ValidateUtil.validate was already being called, I only wrapped it in a try catch block to return the errors to the webservices client. In other words, it seems it was already being called twice.